packages/python/plotly/test_requirements/requirements_310_core.txt

```diff
@@ -1,3 +1,4 @@
 requests==2.25.1
 tenacity==6.2.0
 pytest==7.4.4
+numpy==1.23.2
```

Comment:
I am wondering why we need to adjust these requirements?
Don't we have other tests that load `numpy`?
If so, can't we move these tests to run with those together?

Reply:
Oh yeah the tests that use numpy normally are in `optional`. Do you want me to move it there?